### ISSUE TITLE
Release date estimation?

### ISSUE THREAD

**Issue by Hrxn** [gpfjeff/winhasher#10]
I see some commit activity, but yet...

> Current Release    1.6.2.67
> Date of Release     August 6, 2013

When can we expect a release of a new version, roughly?

Winhasher has always been a must-have utility for me ;-)

**Comment by gpfjeff** [comment]
I was wondering if anyone had been watching... :)

Unfortunately, no, I don't have any good estimates just yet. After I moved the project from Google Code to GitHub, I did try to take some time and make a few updates, such as looking at resolving some of the outstanding issues and (hopefully) prepping the code for additional hashing algorithms (like SHA-3) once the Bouncy Castle folks push out their updates. However, I still don't have as much free time to tackle side projects anymore, so I've only had a chance to do these updates in stolen batches. (Google forced me into the GitHub migration with a deadline.)

I'll make sure to update the README page as soon as I have an idea of a date I can commit to.

I'm planning to bring over to WinHasher the update notification I added to Cryptnos, so once the next version finally does comes out, it should be easier to find out about updates. (Provided, of course, people find out about the update with the update notifications....)
